action.yml

```diff
@@ -4,6 +4,9 @@ branding:
   icon: life-buoy
   color: orange
 inputs:
+  github-token:
+    description: 'GitHub token with permissions look up commit and pull request details'
```

Comment:
```suggestion
    description: 'Optional: GitHub token with permissions to look up commit and pull request details so that CI_COMMIT_MESSAGE is populated with the HEAD commit message'
```